src/aggregation/agg_result.rs

```diff
@@ -18,55 +17,77 @@ use super::intermediate_agg_result::{
 };
 use super::metric::{SingleMetricResult, Stats};
 use super::Key;
+use crate::TantivyError;
 
 #[derive(Clone, Default, Debug, PartialEq, Serialize, Deserialize)]
 /// The final aggegation result.
 pub struct AggregationResults(pub HashMap<String, AggregationResult>);
 
 impl AggregationResults {
+    pub(crate) fn get_value_from_aggregation(
+        &self,
+        name: &str,
+        agg_property: &str,
+    ) -> crate::Result<Option<f64>> {
+        if let Some(agg) = self.0.get(name) {
+            agg.get_value_from_aggregation(name, agg_property)
+        } else {
+            // Should return an error here? Missing aggregation could be intentional to save
```

Comment:
you tell me :D

Reply:
Currently this can't happen. We do validation on the request for this.

In the future it could be intentional or a bug, so I'm not sure. Maybe be strict?

Reply:
I changed it to return an error